lighthouse-core/test/network-records-to-devtools-log.js

```diff
@@ -97,11 +110,11 @@ function getResponseReceivedEvent(networkRecord, index) {
         url: networkRecord.url || exampleUrl,
         status: networkRecord.statusCode || 200,
         headers,
-        mimeType: networkRecord.mimeType || 'text/html',
+        mimeType: typeof networkRecord.mimeType === 'string' ? networkRecord.mimeType : 'text/html',
```

Comment:
I was alarmed that non-strings were making it here, but is this to avoid `''` being falsy?

Reply:
ya pretty much